.cbmc-batch/jobs/aws_cryptosdk_keyring_on_decrypt/aws_cryptosdk_keyring_on_decrypt_harness.c

```diff
@@ -82,6 +82,7 @@ void aws_cryptosdk_keyring_on_decrypt_harness() {
                                                      .on_decrypt = nondet_bool() ? NULL : on_decrypt };
     struct aws_cryptosdk_keyring keyring;
     ensure_cryptosdk_keyring_has_allocated_members(&keyring, &vtable);
+    __CPROVER_assume(keyring.refcount.value != NULL);
```

Comment:
Why not assume the invariant here? What is the equivalence precondition in the harness?

Reply:
This assumption is actually no longer needed. It was a weird property of a stub we were using but we can actually use the real function so I've updated to do that. The aws_cryptosdk_keyring_is_valid invariant is enough and the condition in the harness matches.